ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -0,0 +1,323 @@
+									ic := decredmaterial.NewIcon(pg.Icons.ImageBrightness1)
+									ic.Color = pg.Theme.Color.Gray1
+									return layout.Inset{Top: values.MarginPadding16, Left: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+										return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
+											layout.Rigid(txt.Layout),
+											layout.Rigid(func(gtx C) D {
+												return layout.Inset{Top: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+													return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+														layout.Rigid(func(gtx C) D {
+															return layout.Inset{Bottom: values.MarginPadding12}.Layout(gtx, func(gtx C) D {
+																return ic.Layout(gtx, values.MarginPadding8)
+															})
+														}),
+														layout.Rigid(func(gtx C) D {
+															txt2 := `<span style="text-color: grayText2">
+														<b>Mixed </b> account will be the outbounding spending account.
+													</span>`
+
+															return layout.Inset{
+																Left: values.MarginPadding8,
+															}.Layout(gtx, renderers.RenderHTML(txt2, pg.Theme).Layout)
+														}),
+													)
+												})
+											}),
+											layout.Rigid(func(gtx C) D {
+												return layout.Inset{Top: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+													return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+														layout.Rigid(func(gtx C) D {
+															return layout.Inset{Bottom: values.MarginPadding12}.Layout(gtx, func(gtx C) D {
+																return ic.Layout(gtx, values.MarginPadding8)
+															})
+														}),
+														layout.Rigid(func(gtx C) D {
+															txt3 := `<span style="text-color: grayText2">
+													<b>Unmixed </b> account will be the change handling account.
+												</span>`
+
+															return layout.Inset{
+																Left: values.MarginPadding8,
+															}.Layout(gtx, renderers.RenderHTML(txt3, pg.Theme).Layout)
+														}),
+													)
+												})
+											}),
+										)
+									})
+								}),
+							)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.autoSetupClickable.Layout(gtx, pg.autoSetupLayout)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.manualSetupClickable.Layout(gtx, pg.manualSetupLayout)
+						}),
+					)
+				})
+			},
+		}
+		return page.Layout(gtx)
+	}
+
+	return components.UniformPadding(gtx, body)
+}
+
+func (pg *SetupMixerAccountsPage) autoSetupLayout(gtx C) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
```

Comment:
These can be merged into one.
```suggestion
		return layout.Flex{
		  Axis: layout.Vertical, 
		  Alignment: layout.Middle,
		  Spacing: layout.SpaceBetween,
		}.Layout(gtx,
```

Reply:
![Screenshot from 2022-03-21 15-41-51](https://user-images.githubusercontent.com/25265396/159285350-ab657548-45c5-4f5c-8275-428cafa1dd2e.png)
This breaks the UI

Reply:
will send a temp commit to you.

Reply:
remove the `Axis: layout.Verical`